iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -104,7 +106,7 @@
 public class HiveIcebergStorageHandler implements HiveStoragePredicateHandler, HiveStorageHandler {
   private static final Logger LOG = LoggerFactory.getLogger(HiveIcebergStorageHandler.class);
 
-  private static final String ICEBERG_URI_PREFIX = "iceberg://";
+  private static final String ICEBERG_URI_PREFIX = "iceberg:/";
```

Comment:
Was this change necessary? I think it's supposed to work like a 'protocol' that's why there is the double-slash. I think kafka and druid did the same thing (i.e. `druid://` and `kafka://`)